### ISSUE TITLE
Connect Bank Account - An error message appears 'The validation code you entered is incorrect, please try again.' after navigating back to 'Connect bank account, also the area is grayed out 

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#32742]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.10.0
**Reproducible in staging?:** y
**Reproducible in production?:** not able to check production
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

**Issue found when executing PR** https://github.com/Expensify/App/pull/32205

## Action Performed:
1. Ensure you're connected with staging server
2. Go to workspace > Add Bank Account
3. Click Connect Manually, Routing Number: 011401533, Account number: 1111222233331111 > Continue
I next step fill out Legal Business Name: Alberta Bobbeth Charleson, Tax ID number: 123456789 (any nine digits), addres must be US based and valid, any date > Save & Continue
4. In next step First name: Alberta, Last name: Charleson, Toggle confirmation you are authorized to use company bank > Save & Continue
In Beneficial Owners Toggle terms and conditions, Toggle confirmation that the information provided is true and accurate
5. Enter any number for the fields
5. Back to to Connect bank account.

## Expected Result:
User should not see any error message 'The validation code you entered is incorrect, please try again' after navigating back to 'Connect bank account and area should be greyed out

## Actual Result:
 An error message appears 'The validation code you entered is incorrect, please try again.' after navigating back to 'Connect bank account, also the area is grayed out 

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/0c422ab7-a6e9-49c3-84da-06fdb8108baa

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by deetergp** [comment]
I can reproduce locally as well. I suspect that this PR is our culprit https://github.com/Expensify/App/pull/32205

**Comment by deetergp** [comment]
Okay, I'm not entirely sure that https://github.com/Expensify/App/pull/32205 is the culprit here, but I can say that I tested this locally and was able to recreate the issue. When I checked the linked issue, the testing steps stated that one needed to check the validation amounts in the db, which I did and was able to proceed with validating the account. 

I'm not sure how the Alberta Bobbeth Charleson account is supposed to work but I don't think real users will be affected by this issue, so I'm going to demote from being a blocker.
